AndroidClient/app/build.gradle

```diff
@@ -19,51 +19,72 @@ android {
 
     defaultConfig {
         applicationId "com.pixtory.app"
-        minSdkVersion 11
+        minSdkVersion 16
         targetSdkVersion 23
-        versionCode 1
-        versionName "1.0"
+        versionCode 13 //increment with every release
+        versionName "pre_launch_test_apk" //increment with every release
     }
 
     dexOptions {
         incremental true
         javaMaxHeapSize "4g"
     }
     buildTypes {
+
+        debug{
+            proguardFiles getDefaultProguardFile('proguard-android.txt'),
+                    'proguard-rules.pro'
+        }
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android.txt'),
+                    'proguard-rules.pro'
         }
     }
 }
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:appcompat-v7:23.1.1'
     testCompile 'junit:junit:4.12'
     compile project(':lib_viewpager_indicator')
+
+    //Multidex support
+
+    //bandwidth tracking
+
+    //Amplitude analytics
+    compile 'com.android.support:support-v4:23.4.0'
+    compile 'com.android.support:appcompat-v7:23.4.0'
     compile 'com.google.code.gson:gson:2.3'
     compile 'com.jakewharton:butterknife:7.0.1'
     compile 'com.squareup.retrofit:retrofit:1.9.0'
     compile 'com.facebook.stetho:stetho-okhttp:1.1.1'
-    compile 'com.android.support:recyclerview-v7:22.2.0'
-    compile 'com.android.support:cardview-v7:22.0.0'
+    compile 'com.android.support:recyclerview-v7:23.4.0'
+    compile 'com.android.support:cardview-v7:23.4.0'
     compile 'com.facebook.fresco:fresco:0.7.0'
```

Comment:
Yes mostly fresco, in few places for loading wallpaper we have used
Picasso, which needs to be changed to fresco.

On Jul 20, 2016 4:06 PM, "aasha" <notifications@github.com> wrote:

In AndroidClient/app/build.gradle
<https://github.com/aasha/PhotoStory/pull/3#discussion_r71502142>:

>      compile 'com.google.code.gson:gson:2.3'
>      compile 'com.jakewharton:butterknife:7.0.1'
>      compile 'com.squareup.retrofit:retrofit:1.9.0'
>      compile 'com.facebook.stetho:stetho-okhttp:1.1.1'
> -    compile 'com.android.support:recyclerview-v7:22.2.0'
> -    compile 'com.android.support:cardview-v7:22.0.0'
> +    compile 'com.android.support:recyclerview-v7:23.4.0'
> +    compile 'com.android.support:cardview-v7:23.4.0'
>      compile 'com.facebook.fresco:fresco:0.7.0'

Are you using both fresco and picasso?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/aasha/PhotoStory/pull/3/files/445f5ba6204dec011ae18858a0e52936f28f0644#r71502142>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/ADf9aRCpfPNq0h-4xED3CirFGItVLpvbks5qXfpEgaJpZM4I-RO1>
.